dffml/operation/model.py

```diff
@@ -36,10 +36,10 @@ def __post_init__(self):
 async def model_predict(self, features: Dict[str, Any]) -> Dict[str, Any]:
     async def repos():
         yield Repo("", data={"features": features})
-
+    target_name = self.config.model.config.predict.NAME
     async for repo in self.mctx.predict(repos()):
         return {
             "prediction": {
```

Comment:
This should just be
```python
return {
    "prediction": { repo.prediction() }
}
```

Reply:
since `repo.prediction()` will now have the feature names as the first level keys in their `dict`

Reply:
did you mean `repo.predictions()` after we adopt `feature/features` style

Reply:
Yes, sorry. I had commented this comment before that comment. You are correct.

Reply:
Okay.Thanks